website/content/docs/internals/telemetry/enable-telemetry.mdx

```diff
@@ -0,0 +1,93 @@
+---
+layout: docs
+page_title: Enable Vault telemetry
+description: >-
+  Step-by-step guide to enabling telemetry gathering with Vault
+---
+
+# Enable Vault telemetry gathering
+
+Collect telemetry data from your Vault installation.
+
+## Before you start
+
+- **You must have Vault 1.14 or later installed and running**.
+- **You must have access to your [Vault configuration](/vault/docs/configuration) file**.
+
+
+## Step 1: Choose an aggregation agent
+
+@include 'telemetry/supported-aggregation-agents.mdx'
+
+## Step 2: Enable at least one audit device
+
+To include audit-related metrics, you must enable auditing on at least device
+with the `vault audit enable` command. For example, to enable auditing for the
+`file` device and save the logs to `/var/log/vault_audit.log`:
+
+```shell-session
+$ vault audit enable file file_path=/var/log/vault_audit.log
+```
+
+By default, Vault replicates audit device configuration to all nodes within a
+cluster. Enterprise installations also replicate audit device configuration to
+the primary node and all secondaries in a cluster. To disable replication, use
+the `local` parameter:
+
+```shell-session
+$ vault audit enable file -local file_path=/var/log/vault_audit.log
+```
+
+
+## Step 3: Configure telemetry collection
+
+To configure telemetry collection, update the telemetry stanza in your Vault
+configuration with your collection preferences and aggregation agent details.
+
+For example, the following `telemetry` stanza configures Vault with the standard
+telemetry defaults and connects it to a Statsite agent running on the default
+port within a company intranet at `mycompany.statsite`:
+
+```json
+telemetry {
+  usage_gauge_period = "10m",
```

Comment:
These commas don't belong in HCL config I think, this looks like a mix of JSON and HCL formatting.

Reply:
The syntax should be fixed.